services/retrieve_and_rank/v1.js

```diff
@@ -51,20 +56,20 @@ module.exports = function (RED) {
         var params = {
           training_data: msg.payload
         };
-      
+
         if (config.rankername) {
-          params.training_metadata = "{\"name\": \""+config.rankername+"\"}";
+          params.training_metadata = '{\"name\": \"'+config.rankername+'\"}';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'training_metadata' is not in camel case.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2690663401/issues/source?bid=3330694&fileBranchId=3336646#l61)